otx/cli/builder/supported_backbone/mmcls.json

```diff
@@ -1,5 +1,154 @@
 {
   "backbones": {
+    "mmcls.AlexNet": {
+      "required": [],
+      "options": {},
+      "available": []
+    },
+    "mmcls.Conformer": {
+      "required": ["arch"],
+      "options": {
+        "arch": ["tiny", "small", "base"]
+      },
+      "available": []
+    },
+    "mmcls.ConvMixer": {
+      "required": ["arch"],
+      "options": {
+        "arch": ["768/32", "1024/20", "1536/20"]
+      },
+      "available": []
+    },
+    "mmcls.ConvNeXt": {
+      "required": ["arch"],
+      "options": {
+        "arch": ["tiny", "small", "base", "large", "xlarge"]
+      },
+      "available": []
+    },
+    "mmcls.CSPDarkNet": {
+      "required": ["depth"],
+      "options": {
+        "depth": [53]
+      },
+      "available": []
+    },
+    "mmcls.CSPResNet": {
+      "required": ["depth"],
+      "options": {
+        "depth": [50]
+      },
+      "available": []
+    },
+    "mmcls.CSPResNeXt": {
+      "required": ["depth"],
+      "options": {
+        "depth": [50]
+      },
+      "available": []
+    },
+    "mmcls.DistilledVisionTransformer": {
+      "required": [],
+      "options": {},
+      "available": []
+    },
+    "mmcls.DenseNet": {
+      "required": ["arch"],
+      "options": {
+        "arch": ["121", "169", "201", "161"]
```

Comment:
Is this right order?

Reply:
I think the order is not important, but I set the first value as default. (This value is also used as default in official mmX.)